src/client/src/commons/form/borehole/boreholeForm.js

```diff
@@ -501,6 +501,7 @@ class BoreholeForm extends React.Component {
                       this.props.user.data.username
                   )
                 }
+                t={this.props.t}
```

Comment:
REQ: Die Übersetzungsfunktion sollte nicht als prop übergeben werden sondern in der `EditorBoreholeFilesTable` Componente mithilfe von `withTranslation`. (bzw. `useTranslation` bei neuen Function Components)